sdk/tables/data-tables/test/public/transaction.spec.ts

```diff
@@ -19,11 +19,58 @@ const testEntities = [
   { partitionKey, rowKey: "3", name: "third" },
 ];
 
+const suffix = isNode ? "node" : "browser";
+
+describe("concurrent batch operations", () => {
+  const concurrentTableName = `concurrentBatchTableTest${suffix}`;
+  let unRecordedClient: TableClient;
+  before(async () => {
+    if (!isPlaybackMode()) {
+      unRecordedClient = await createTableClient(concurrentTableName, "SASConnectionString");
+      await unRecordedClient.createTable();
+    }
+  });
+
+  after(async () => {
+    if (!isPlaybackMode()) {
+      await unRecordedClient.deleteTable();
+    }
+  });
+  beforeEach(async function (this: Context) {
+    sinon.stub(Uuid, "generateUuid").returns("fakeId");
+    unRecordedClient = await createTableClient(concurrentTableName, "SASConnectionString");
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+  });
+
+  it("should send concurrent transactions", async () => {
+    // Only run this in live mode. Enable playback when https://github.com/Azure/azure-sdk-for-js/issues/24189 is fixed
+    if (!isLiveMode()) {
+      return;
```

Comment:
instead of `return` can do `this.skip()` if you switch away from arrow functions. That will help mocha correctly track that it was skipped and not trivially passed.